### PR TITLE
docs(client_libs): update code examples for v3 client to new unified API

### PR DESCRIPTION
**Pull request by bednar**
Updated code examples for v3 clients to new unified API. For more info see changes in releases notes:

C# - 0.2.0 - https://github.com/influxcommunity/influxdb3-csharp/blob/main/CHANGELOG.md#020-2023-08-11
Java/JVM - 0.2.0 - https://github.com/InfluxCommunity/influxdb3-java/blob/main/CHANGELOG.md#020-2023-08-11
GoLang - 0.2.0 - https://github.com/InfluxCommunity/influxdb3-go/blob/main/CHANGELOG.md#020-2023-08-11

### Checklist

Authors and Reviewer(s), please verify the following:

- [x] A PR description, regardless of the triviality of this change, that communicates the value of this PR
- [x] [Well-formatted conventional commit messages](https://www.conventionalcommits.org/en/v1.0.0/) that provide context into the change
- [x] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: src/homepageExperience/components/steps/go/ExecuteQuerySql.tsx [hunk 1]
```diff
@@ -33,7 +27,10 @@ query := \`SELECT *
           WHERE time >= now() - interval '1 hour'
             AND ('bees' IS NOT NULL OR 'ants' IS NOT NULL)\`
 
-iterator, err := client.Query(context.Background(), "${bucket}", query)
+queryOptions := influxdb3.QueryOptions{
+  Database: database,
```
Comment (mavarius) [inline src/homepageExperience/components/steps/go/ExecuteQuerySql.tsx:31]: I see that `bucket` is being removed but then how is this `database` variable populated?

File: src/homepageExperience/containers/GoWizard.tsx [hunk 2]
```diff
@@ -154,7 +154,7 @@ export class GoWizard extends PureComponent<null, State> {
         return <WriteDataSql onSelectBucket={this.handleSelectBucket} />
       }
       case 5: {
-        return <ExecuteQuerySql bucket={this.state.selectedBucket} />
+        return <ExecuteQuerySql />
```
Comment (mavarius) [inline src/homepageExperience/containers/GoWizard.tsx:157]: Are we sure we don't need to populate `database` with the `bucket` here?

**Review by mavarius — COMMENTED** [review]
My only comment is on the removal of bucket. It was populating the database name in the snippet, if it's removed, is it now just the generic `database`?

File: src/homepageExperience/components/steps/go/ExecuteQuerySql.tsx (hunk 1 above)
Comment (bednar) [reply src/homepageExperience/components/steps/go/ExecuteQuerySql.tsx:31]: The `database` is initialised in the "write-data" step: https://github.com/influxdata/ui/blob/a60db0a94c077ff49751490115a109cbc1df0400/src/homepageExperience/components/steps/go/WriteDataSql.tsx#L95

File: src/homepageExperience/containers/GoWizard.tsx (hunk 2 above)
Comment (bednar) [reply src/homepageExperience/containers/GoWizard.tsx:157]: Yes, the`database` is initialised in the "write-data" step: https://github.com/influxdata/ui/blob/a60db0a94c077ff49751490115a109cbc1df0400/src/homepageExperience/components/steps/go/WriteDataSql.tsx#L95

**Review by mavarius — APPROVED** [review]
good to go

File: src/homepageExperience/components/steps/go/WriteDataSql.tsx [hunk 3]
```diff
@@ -76,16 +76,16 @@ func main() {
     token := os.Getenv("INFLUXDB_TOKEN")
 
     // Create a new client using an InfluxDB server base URL and an authentication token
-    client, err := influx.New(influx.Configs{
-      HostURL: url,
-      AuthToken: token,
+    client, err := influxdb3.New(influxdb3.ClientConfig{
+      Host:  url,
+      Token: token,
```
Comment (mfreeman451) [inline src/homepageExperience/components/steps/go/WriteDataSql.tsx:81]: Should also have database here

File: src/homepageExperience/components/steps/go/WriteDataSql.tsx [hunk 4]
```diff
@@ -76,16 +76,16 @@ func main() {
     token := os.Getenv("INFLUXDB_TOKEN")
```
Comment (mfreeman451) [inline src/homepageExperience/components/steps/go/WriteDataSql.tsx:77]: database := os.Getenv("INFLUXDB_DATABASE")

**Pull request merged**